.drone.yml

```diff
@@ -1,16 +1,40 @@
 ---
 kind: pipeline
-name: default
+name: testing
 
 platform:
   os: linux
   arch: amd64
 
 steps:
+- name: vet
+  pull: always
+  image: golang:1.11
+  commands:
+  - go vet ./...
```

Comment:
starting with Go 1.10 the `go test` command automatically runs the `vet` command

Reply:
Doesn't seem to be the case, if I add something `vet` complains about the `test` is running through totally fine on Go 1.11.

Reply:
hmm strange, I remembered reading in the release notes that `vet` was run during the testing phase. https://golang.org/doc/go1.10#test  Do you know if there are exceptions?

Reply:
ah ok, it looks like only a subset of the go vet checks are run by go test:

> Only a high-confidence subset of the default go vet checks are used. That subset is: 'atomic', 'bool', 'buildtags', 'nilfunc', and 'printf'.